ext/webextension/src/browser_action/main_popup.js

```diff
@@ -509,4 +509,34 @@ document.querySelector('#siteconfig_show').addEventListener('click', function()
     ui.hide('#siteconfig_show');
 });
 
+function indetifierUpdate() {
+    let text = document.querySelector("#username").value;
+    text += document.querySelector('#masterkey').value;
+    let identifier = document.querySelector('#identifier');
+    if (text == '' || text == null || text.length < 6)
+        identifier.innerHTML = '';
+    else if (text.length >=6) {
+        let emojiStr = "&#1285";
+        let a = 0, b = 0;
+        let isA = true;
+        for (let i = 0; i < text.length; i++) {
+            if (isA)
+                a += text.charCodeAt(i);
+            else 
+                b += text.charCodeAt(i);
+            isA = !isA;
+        }
+        a = a % 90;
```

Comment:
I think there is a bug here
First, the emoji set only has 0x50 (80) symbols, not 90, so should probably `% 80`, not 90
Second, you're only adding the offset (+12) for to the start of the range, so both a=0 and a=12 will give: 😀 - this will, however, save you from overshooting the range (90 vs 80)


It might be what you where targeting, but it looks kind'a strange.

Reply:
that makes sense, guess I didn't consider that scenario. I was going for the range between 12 and 90 because outside that range I found some black color emoji that won't be visible because of our dark background.
I have raised new PR with % 78 and added 12 every time so that range will be between 12 and 90. 
Also added space between emojis which make it look better.